### ISSUE TITLE
Don't work

### ISSUE THREAD

**Issue by deevroman** [AlessandroZ/LaZagne#134]
```
Traceback (most recent call last):
File "laZagne.py", line 390, in <module>
File "laZagne.py", line 239, in runLaZagne
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd0 in position 0: ordinal not in range(128)
Failed to execute script laZagne
```

**Comment by A888R** [comment]
Yep, I see the same error

**Comment by AlessandroZ** [comment]
I need more information to better understand the problem. Which language do you use ? What's the module launch ? (I suppose all modules should have this problem). 

Thank you.

**Comment by AlessandroZ** [comment]
It should be fixed on the next release 2.3. The code has been pushed but the executable has not been built yet. 
If some values are not well printed on the console, try to write all passwords on a file, everything should be well printed (I have tried with russian, and asian characters). 
`lazagne.exe all -oA `
